access.go

```diff
@@ -567,6 +571,55 @@ func (s *Server) handleClientCredentialsRequest(w *Response, r *http.Request) *A
 	return ret
 }
 
+func (s *Server) handleLoginQueueTicketRequest(w *Response, r *http.Request) *AccessRequest {
+	auth, err := CheckBasicAuth(r)
+	if err != nil {
+		w.SetError(E_SERVER_ERROR, err.Error())
+		w.InternalError = err
+		return nil
+	}
+
+	var clientID string
+	var client Client
+	if auth == nil {
+		clientID = r.Form.Get("client_id")
+		if clientID == "" {
+			w.SetError(E_UNAUTHORIZED_CLIENT, "missing client_id in form body")
+			return nil
+		}
+		client = getClientWithoutSecret(clientID, w.Storage, w)
+	} else {
+		// get client authentication
+		auth := GetClientAuth(w, r, s.Config.AllowClientSecretInParams)
+		if auth == nil {
+			return nil
+		}
+		client = getClient(auth, w.Storage, w)
+	}
+
+	// generate access token
+	ret := &AccessRequest{
+		Type:            EXTEND_LOGIN_QUEUE_TICKET,
+		Code:            r.Form.Get("login_queue_ticket"),
+		CodeVerifier:    r.Form.Get("code_verifier"),
```

Comment:
will it have code_verify here?

Reply:
I just keep the param here in case we need to implement the code challenge & code verifier validation in login queue service